docs/resources/howto/wiki.md

```diff
@@ -0,0 +1,273 @@
+
+### Adding a page
+
+In order to add a new page, a new `.md` should be created in the appropriate location in the folder structure.
+A link to the page then needs to be added in the `nav` section of the `mkdocs.yml`, together with an ID.
+
+## Guidelines
+
+Pages are written in Markdown (file extension `.md`).
+A general overview of the syntax a well as some best practices can be found [here][markdownsyntax]{target=_blank}.
+Additionally, to allow for easier comparison between two versions of a file, it is recommended to keep it to one sentence per line.
+Ideally, the line length is kept below 100 characters.
+Following this, it is also recommended to not put links in the text, instead creating an ID at the end of the document and linking to this.
+
+Different blocks of either code or text should be separated by one blank line.
+
+To create blocks of code, use `fenced code blocks`, which are created using `` ``` ``.
+These blocks of code should be separated from the previous and following text by one blank line.
+To allow for syntax highlighting, the language should be specified.
+Below a basic example.
+
+````markdown
+```bash
+something code that does something
+```
+````
+
+## Environments
+
+### General text formatting
+
+These are basic markdown commands, repeated here for convenience.
+A more exhaustive list is available [here][markdownsyntax].
+
+To create a **bold** text, surround the text with `**text**`.
+
+To create an __italic__ text, surround the text with `__text__`.
+
+To create a [link][bestwiki]{target=_blank}, use `[link][bestwiki]{target=_blank}`.
+Note that at the bottom of the file, an ID named `bestwiki` should be created, together with the hyperlink to the webpage.
+The specifier `{target=_blank}` is added to ensure pages open in a new tab.
+In the specifier, additional information can be added if the link is on the accessibility of the link
+Links accessable only with a CERN login can be marked like `{target=_blank .cern_login}` or from the CERN network like `{target=_blank .cern_internal}`.
+
+To add small hints to a difficult word, which appear on mouse over, add `*[difficult word]: helpful explanation` at the bottom of the file.
+
+### Code listing
+
+To highlight `code` inline, surround the text to highlight with `` ` ``.
+For creating a code block, surround the code with `` ``` ``.
+Please make sure that the code block is separated from the text by one blank line.
+Additionally, please specify the language.
+
+````markdown
+```python
+for i in range(3):
+    print(i)
+```
+````
+
+### Lists
+
+#### Unordered list
+
+An unordered list looks like this.
+
+- First
+- Second
+- Third
+
+It can be created using
+
+```markdown
+- First
+- Second
+- Third
+```
+
+#### Ordered list
+
+An ordered list looks like this.
+
+1. First
+2. Second
+3. Third
+
+It can be created using
+
+```markdown
+1. First
+2. Second
+3. Third
+```
+
+#### Task list
+
+A task list looks like this.
```

Comment:
Making a tasklist with hints, as it is used in the procedures, is not possible in pure markdown and we need to resort to some `html` code. 

```html
- [ ] <details class="nodeco"><summary>Task 1 Summary</summary>
      <p> Hints for task 1.
      </p></details>

- [ ] <details class="nodeco"><summary>Task 2 Summary</summary>
      <p> Hints for task 2.
      </p></details>

```